src/Main.ts

```diff
@@ -64,6 +64,8 @@ export interface ISlackTeam {
 
 interface MetricsLabels { [labelName: string]: string; }
 
+type TimerFunc = (labels?: Partial<Record<string, string | number>> | undefined) => void;
```

Comment:
What does `Partial` give us here?
I think `Partial<Record<string, string | number>>` is the same as `Record<string, string | number>` as a parameter when you're not using any other property in the function.

Reply:
I copy pasted it from the library :shrug: